### PR TITLE
update(heading): Adjust naming of headers to match Helix-ui

### PR DESCRIPTION
**Pull request by technabors**
Headers now match Helix-ui naming specification. Additionally h5 styles
were written in CSS and added the `_shame.scss` file. These can be
removed once we update to the newest version of helix-ui; The version
being used currently does not include these styles.

### CODE REVIEW

File: assets/css/_sass/website/_shame.scss [hunk 1]
```diff
@@ -5,4 +5,14 @@
   thead th {
     padding-top: 0;
   }
+}
+
+/* 
+* This will be removed when we move to version X.X (laura help me here) of the helix-ui code. The version currently used on the site does not have styling for the hxHeading-5 class.
```
Comment (technabors) [inline assets/css/_sass/website/_shame.scss]: @nimbinatus I was unclear what version of the Helix-ui toolkit we will need to have access to the new header class names. This comment is basically placeholder to explain why I was adding styling for the time being for H5's.

File: _docs/style/typography.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ helix-ui-javascript: false
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Page title
+### Heading one
 
-Page titles are the highest level of hierarchy. There should only be one on any page.
+Heading one's are the highest level of hierarchy. There should only be one on any page.
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`one's`/`ones`

Comment (nimbinatus) [reply _docs/style/typography.md]: Same for all below

File: _docs/style/typography.md [hunk 3]
```diff
@@ -51,14 +51,14 @@ Page titles are the highest level of hierarchy. There should only be one on any
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Section title
+### Heading two
 
-Use section titles for titles of page sections. For example, if your page title is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading uses section titles.
+Use heading two's for titles of page sections. For example, if your Heading one is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading uses the heading two style.
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`Heading one`/`heading one`

File: assets/css/_sass/website/_shame.scss (hunk 1 above)
Comment (nimbinatus) [reply assets/css/_sass/website/_shame.scss]: h5 appeared in v0.3.0+; we'll be moving to the latest, which is currently v0.4.1

File: assets/css/_sass/website/_shame.scss [hunk 4]
```diff
@@ -5,4 +5,14 @@
...
     padding-top: 0;
   }
+}
+
+/* 
+* This will be removed when we move to version X.X (laura help me here) of the helix-ui code. The version currently used on the site does not have styling for the hxHeading-5 class. 
+*/
+
+.hxHeading5 {
+	font-size: 0.75rem;
```
Comment (nimbinatus) [inline assets/css/_sass/website/_shame.scss]: Spacing is a bit weird here. Check indent.

File: _docs/style/typography.md [hunk 5]
```diff
@@ -51,14 +51,14 @@ Page titles are the highest level of hierarchy. There should only be one on any
...
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<h2 class="hxSectionTitle">Section title</h2>' typeface="Roboto" weight="Light (300)" fontsize="30px / 1.875rem" csscode="Gray 900" hex="#424242" linespace="1" bottomspace="28px / 1.75rem" %}
+{% include type-styles.html htmlexample='<h2 class="hxSectionTitle">Heading two</h2>' typeface="Roboto" weight="Light (300)" fontsize="32px / 1.875rem" csscode="Gray 900" hex="#424242" linespace="1" bottomspace="28px / 1.75rem" %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated

File: _docs/style/typography.md (hunk 2 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 3 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: assets/css/_sass/website/_shame.scss (hunk 4 above)
Comment (technabors) [reply assets/css/_sass/website/_shame.scss]: updated

File: _docs/style/typography.md [hunk 6]
```diff
@@ -51,14 +51,14 @@ Page titles are the highest level of hierarchy. There should only be one on any
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Section title
+### Heading two
 
-Use section titles for titles of page sections. For example, if your page title is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading uses section titles.
+Use heading two's for titles of page sections. For example, if your heading one is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading uses the heading two style.
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`two's`/`twos`

Comment (technabors) [reply _docs/style/typography.md]: fixed

**Pull request merged**